include/deal.II/cgal/utilities.h

```diff
@@ -610,9 +610,10 @@ namespace CGALWrappers
   resort_dealii_vertices_to_cgal_order(const unsigned int            structdim,
                                        std::vector<Point<spacedim>> &vertices)
   {
```

Comment:
Do we need
```suggestion
  resort_dealii_vertices_to_cgal_order([[maybe_unused]] const unsigned int            structdim,
                                       [[maybe_unused]] std::vector<Point<spacedim>> &vertices)
  {
```
?

Reply:
@masterleinad Good point. Not according to the standard because an `if constexpr (...)` does still imply (unconditional) usage. But some faulty versions of gcc/clang issue a warning and we will likely need to work around this in a follow-up PR.

@bangerth Would you mind to mark the parameters as potentially unused (either the new or the old way)?